cloudify_agent/tests/__init__.py

```diff
@@ -66,15 +66,24 @@ def get_local_cert_path(temp_folder):
         return f.name
 
     @staticmethod
+    def _clean_cert(cert_content):
+        """ Strip any whitespaces, and normalize the string on windows """
+
+        cert_content = cert_content.strip()
+        cert_content = cert_content.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
i think simply opening the file with universal newlines ('U') mode would do this as well?